actionpack/lib/action_dispatch/routing/mapper.rb

```diff
@@ -400,9 +400,24 @@ module Base
         # You should put the root route at the top of <tt>config/routes.rb</tt>,
         # because this means it will be matched first. As this is the most popular route
         # of most Rails applications, this is beneficial.
-        def root(options = {})
-          name = has_named_route?(:root) ? nil : :root
-          match '/', { as: name, via:  :get }.merge!(options)
+        def root(path, options = {})
```

Comment:
Can we move the methods to `Resources`? That way `scope` would be already defined.